tc-tests-utils.sh

```diff
@@ -636,22 +651,42 @@ package_native_client()
     echo "Please specify artifact name."
   fi;
 
-  if [ -f "${tensorflow_dir}/bazel-bin/native_client/libdeepspeech_model.so" ]; then
-    tar -cf - \
-      -C ${tensorflow_dir}/bazel-bin/native_client/ generate_trie \
-      -C ${tensorflow_dir}/bazel-bin/native_client/ libdeepspeech.so \
-      -C ${tensorflow_dir}/bazel-bin/native_client/ libdeepspeech_model.so \
-      -C ${deepspeech_dir}/ LICENSE \
-      -C ${deepspeech_dir}/native_client/ deepspeech \
-      -C ${deepspeech_dir}/native_client/kenlm/ README.mozilla \
-      | pixz -9 > "${artifacts_dir}/${artifact_name}"
-  else
-    tar -cf - \
-      -C ${tensorflow_dir}/bazel-bin/native_client/ generate_trie \
-      -C ${tensorflow_dir}/bazel-bin/native_client/ libdeepspeech.so \
-      -C ${deepspeech_dir}/ LICENSE \
-      -C ${deepspeech_dir}/native_client/ deepspeech \
-      -C ${deepspeech_dir}/native_client/kenlm/ README.mozilla \
-      | pixz -9 > "${artifacts_dir}/${artifact_name}"
+  tar -cf - \
+    -C ${tensorflow_dir}/bazel-bin/native_client/ generate_trie \
+    -C ${tensorflow_dir}/bazel-bin/native_client/ libdeepspeech.so \
+    -C ${deepspeech_dir}/ LICENSE \
+    -C ${deepspeech_dir}/native_client/ deepspeech \
+    -C ${deepspeech_dir}/native_client/kenlm/ README.mozilla \
+    | pixz -9 > "${artifacts_dir}/${artifact_name}"
+}
+
+package_native_client_ndk()
```

Comment:
I'm wondering if we should not deal with that in `makefile` as well, like a `dist` target for default `native_client.tar.xz` and a `dist-ndk` for `NDK` builds ? Would maybe make everyone's life easier